### PR TITLE
add fontconfig configuration and add it to installation instructions

### LINKED ISSUE #408: Hack is considered sans-serif by fontconfig by default
fontconfig consider fonts to be sans-serif by default, which breaks font fallback mechanism. This configuration fixes that:

```xml
<alias>
    <family>Hack</family>
    <default><family>monospace</family></default>
</alias>
```

You should either include and distribute this configuration or request a change to default fontconfig configuration.

See [here](https://eev.ee/blog/2015/05/20/i-stared-into-the-fontconfig-and-the-fontconfig-stared-back-at-me/#trial-the-first-monospace-font-falling-back-to-proportional-unicode-glyph) for details.

### PR DESCRIPTION
**Pull request by ariasuni**
Fix #408

### CODE REVIEW

**Comment by chrissimpkins** [comment]
Thanks so much Mélanie!  Really appreciate this contribution to the project. I should be able to review this tonight and get it merged.

**Comment by chrissimpkins** [comment]
Mind rebasing this on our `dev` branch?  This looks great.  Will merge it into dev and this will be added as part of the merge to master for the v3.003 release.  I will add the CHANGELOG update when we get this merged.

File: docs/CONTRIBUTORS.md [hunk 1]
```diff
@@ -92,6 +92,7 @@
 - Shawn Starr - Linux
 - @rahilwazir - Linux
 - Jakub Žitný - NPM
+- Mélanie Chauvel (ariasuni) - Linux
```
Comment (chrissimpkins) [inline docs/CONTRIBUTORS.md]: OK to move your name up the list here.  We use alphabetized contributor lists.

**Comment by ariasuni** [comment]
See #410

**Pull request closed without merging**